trillian/migrillian/core/trillian.go

```diff
@@ -61,12 +60,7 @@ func NewPreorderedLogClient(
 	if got, want := tree.TreeType, trillian.TreeType_PREORDERED_LOG; got != want {
 		return nil, fmt.Errorf("tree %d is %v, want %v", tree.TreeId, got, want)
 	}
-	// If this isn't an RFC6962 hasher, Questions Should Be Asked.
-	logHasher, err := registry.NewLogHasher(tree.HashStrategy)
```

Comment:
Maybe add a check here that `tree.HashStrategy == RFC6962`?

Reply:
Done.